connectors/source-mysql/README.md

```diff
@@ -13,108 +13,98 @@ For example,MySql database vance has table vance_test Look:
+  "iodebeziumdb": "dbname",
+  "iodebeziumfile": "binlog.000009",
+  "iodebeziumpos": "197",
+  "iodebeziumname": "dbname",
+  "iodebeziumtsms": "1671774889000",
+  "iodebeziumtable": "user",
+  "iodebeziumop": "r",
+  "iodebeziumversion": "2.0.1.Final",
+  "iodebeziumrow": "0",
   "data": {
-    "id": 18,
-    "name": "xdl",
-    "description": "Development Manager",
-    "date": "2022-07-06"
+    "after": {
+      "id": 1,
+      "name": "user_name",
+      "email": "user_email"
+    }
   }
 }
 ```
 
 ## MySql Source Configs
 
-Users can specify their configs by either setting environments variables or mount a config.json to
-`/vance/config/config.json` when they run the connector. Find examples of setting configs [here][config].
-
-### Config Fields of the Mysql Source
-
-| name                 | requirement | description                                                                    |
-|----------------------|-------------|--------------------------------------------------------------------------------|
-| host                 | required    | db host                                                                        |
-| port                 | required    | db port                                                                        |
-| username             | optional    | db username                                                                    |
-| password             | optional    | db password                                                                    |
-| db_name              | required    | db database name                                                               |
-| v_target             | required    | target URL will send CloudEvents to                                            |
-| v_store_file         | required    | save offset file name                                                          |
-| db_history_file      | required    | save db schema history file name                                               |
-| store_offset_key     | optional    | offset store use key, default is vance_debezium_offset                         |
-| offset_binlog_file   | optional    | binlog filename, increment sync start binlog file name if not set is full sync |
-| offset_binlog_pos    | optional    | binlog position, use with config offset_binlog_file                            |
-| offset_binlog_gtids  | optional    | binlog grids                                                                   |
+### Config
+
+| name                    | requirement | description                                                                    |
+|-------------------------|-------------|--------------------------------------------------------------------------------|
+| target                  | required    | target URL will send CloudEvents to                                            |
+| db_config.host          | required    | db host                                                                        |
```

Comment:
use mysql.host, mysql.binlog_offset, etc to keep unified style

Reply:
- the host, port... is in the secret object, so it can't merge with binlog_offset
- all the config belong to MySQL, it doesn't need to add MySQL prefix

Reply:
modify to db